### ISSUE TITLE
Stripe Connect: Receiving Transfer webhook from Connect Customer

### ISSUE THREAD

**Issue by kinesii** [pinax/pinax-stripe#524]
#### Issue Summary

Transfers webhooks with Stripe Connect configuration do not parse valid events.

---

---

#### Steps to Reproduce

Receive a 'payout.created' webhook from a Stripe Connect 'connected customer'.

Example webhook request body:

```python
{
  "id": "<event_id>",
  "object": "event",
  "account": "<account_id>",
  "api_version": "2017-08-15",
  "created": 1511725645,
  "data": {
    "object": {
      "id": "<po_id>",
      "object": "payout",
      "amount": 1000,
      "arrival_date": 1511740800,
      "automatic": false,
      "balance_transaction": "<txn_id>",
      "created": 1511725645,
      "currency": "cad",
      "description": "asdasd",
      "destination": "<bank ID>",
      "failure_balance_transaction": null,
      "failure_code": null,
      "failure_message": null,
      "livemode": false,
      "metadata": {
      },
      "method": "standard",
      "source_type": "card",
      "statement_descriptor": "asdas",
      "status": "pending",
      "type": "bank_account"
    }
  },
  "livemode": false,
  "pending_webhooks": 1,
  "request": {
    "id": "<req_id>",
    "idempotency_key": null
  },
  "type": "payout.created"
}
```

#### What were you expecting to happen?

A request record would appear in the pinax_stipe_events table with a valid flag as true.
A transfer record would be created in the pinax_stripe_transfer table.

#### What actually happened?

Request shows in the pinax_stipe_events table, and the 'valid' column is blank/null instead. No errors or exceptions are logged.

**Comment by kinesii** [comment]
I don't really understand, I am using the latest API "You're using API version 2017-08-15" but my API requests from Stripe -> my server are still using `kind = payment.created` instead of `kind = transfer.created` and this is why `WebhookClass = registry.get(kind)` is causing my webhooks not to process.

**Comment by newman99** [comment]
Payouts and transfers are now different things: https://stripe.com/docs/transfer-payout-split .

A transfer moves money from one Stripe account to another.

A payout moves money from a Stripe account to an external account.

**Comment by blueyed** [comment]
There were some fixes in this regard in 4d40903, but it seems to need more fixes apparently.
Please consider creating a PR.

**Comment by paltman** [comment]
Closing as #504 was merged and I don't see a quick resolution here.  Please re-open if still an issue.
